src/app/productConfig/productConfigForm/productConfigForm.tpl.html

```diff
@@ -214,7 +214,7 @@ <h4 class="panel-title border-bottom-small mt0 pt-" translate>
         </div><!-- // panel -->
       </div>
     </div>
-    <div class="panel panel-default give-modal-panel">
+    <div ng-if="$ctrl.useV3 !== 'true'" class="panel panel-default give-modal-panel">
```

Comment:
One concern I realize I have with this is our datadog tests. These rely on the special instructions in order to work. We'll need to make sure that our tests are pointing at a v2 branded checkout.